.gitignore

```diff
@@ -165,7 +165,7 @@ streaming/java/**/.project
 streaming/java/**/*.log
 
 # python virtual env
-venv
+venv*
```

Comment:
Can we undo this change? We probably don't want to match all files that might start with `venv`